### ISSUE TITLE
$support-02 and $inverse-support-02 too low of contrast for text in most themes

### ISSUE THREAD

**Issue by abbeyhrt** [carbon-design-system/carbon#6329]
It seems like for the white and g10 themes, `$support-02` and `$inverse-support-02` fail DAP for color contrast when they're used for regular text.  `$inverse-support-02` also fails for the g90 theme. 

![Screen Shot 2020-06-23 at 12 03 42 PM](https://user-images.githubusercontent.com/17281178/85433666-5e265800-b54a-11ea-84a1-2bb2f36c90c3.png)

![Screen Shot 2020-06-23 at 12 10 09 PM](https://user-images.githubusercontent.com/17281178/85433751-857d2500-b54a-11ea-8769-44b22437c188.png)

It possible that these tokens aren't meant to be used as text so let me know if that's the case!

**Comment by joshblack** [comment]
Hey there @abbeyhrt! 👋 

Only the red support color is ever used with text. To indicate success, I think the recommendation is to use icons. Hope this helps!

**Comment by aagonzales** [comment]
Yes, only error state gets to use colored text. Any other sort of status message should use a `$text-02`

We don't have an official success state for inputs but here are some ways I see it could be done:
![image](https://user-images.githubusercontent.com/11670886/85618455-72d81e00-b626-11ea-9139-e3a622ef3c6f.png)

**Comment by jeanservaas** [comment]
Design options look good @aagonzales — def no green text! 

Also... I meant to revisit the idea of updating $support-02 to Green 60... Would need to deal with that in another issue. but in data viz, a 50 on gray 10, even if it's a graphic is no bueno.

**Comment by abbeyhrt** [comment]
@aagonzales ah okay, that makes sense to me! We'll change the component to reflect that. Thanks for getting back to me! Feel free to close this issue 👍
